gits/terminal.py

```diff
@@ -669,6 +672,10 @@ def dumphtml(self):
             if self._screen[i] & (1 << UNDERLINE_BIT):
                 current_classes.append('underline')
 
+            if self._screen[i] & (1 << REVERSE_BIT):
```

Comment:
Are you sure that this if statement will ever be true?
We clean the `REVERSE_BIT` at the beginning of the `dumphtml` method.